doc/build_wamr.md

```diff
@@ -217,6 +218,11 @@ cmake .. -DWAMR_BUILD_JIT=1
 make
 ```
 
+Moreover, pass arguments `-DWAMR_BUILD_JIT=1` and `-DWAMR_BUILD_LAZY_JIT=1` together to cmake to enable WASM Lazy JIT.
+If Lazy JIT is enabled, then jit function bodies in the module will not be compiled until they are first called,
+so compile time reduces significantly.
```

Comment:
We found that all the AOT function pointers are gotten when loading aot module (aot_loader.c, L2644), seems that the JIT also compiles all the AOT functions firstly, can it reduce a lot compile time now?
We are checking whether we need to refine it in the future, e.g. set all module->func_ptrs to NULL when loading, and require LLVM to compile only when module->func_ptrs[i] is used, e.g. call AOT func from runtime or call indirect from AOT code.